src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -345,7 +345,7 @@ class DetailsSidebar extends React.PureComponent<Props, State> {
                             retentionPolicy={retentionPolicy}
                             bannerPolicy={bannerPolicy}
                             onRetentionPolicyExtendClick={onRetentionPolicyExtendClick}
-                            isLoading={isLoadingAccessStats && isLoadingClassification}
+                            isLoading={isLoadingAccessStats || isLoadingClassification}
```

Comment:
If you are ||ing, then my prior comment applies in your prior patch about just having one isLoading state variable.

Reply:
I just realized that I've been doing this wrong. Access stats *doesn't* have a loading state, so no matter what, it just pops in. For some strange reason, this component has always set `SidebarFileProperties` loading state based off of Access Stats. Should we add a loading crawler to `SidebarAccessStats`?

Reply:
IMO yes

Reply:
Made a ticket or it. Will look next sprint :dance: